Bug: build failure

### Current Behaviour

/home/azero/azerothcore-wotlk/modules/mod-item-level-up/src/LevelItem.cpp:37:10: error: ‘void mod_levelitem_Announce::OnLogin(Player*)’ marked ‘override’, but does not override
   37 |     void OnLogin(Player* player) override
      |          ^~~~~~~
compilation terminated due to -Wfatal-errors.
make[2]: *** [modules/CMakeFiles/modules.dir/build.make:678：modules/CMakeFiles/modules.dir/mod-item-level-up/src/LevelItem.cpp.o] error 1


### Expected Behaviour

build success

### Steps to reproduce the problem

clone 
cmake ...
make 

### Extra Notes

_No response_

### AC rev. hash/commit

last version

### Operating system

ubuntu24.04

### Custom changes or Modules

_No response_

--- a/src/LevelItem.cpp
+++ b/src/LevelItem.cpp
@@ -1,11 +1,11 @@
+#include "AccountMgr.h"
 #include "Chat.h"
 #include "Configuration/Config.h"
-#include "Player.h"
 #include "Creature.h"
-#include "AccountMgr.h"
-#include "ScriptMgr.h"
 #include "Define.h"
 #include "GossipDef.h"
+#include "Player.h"
+#include "ScriptMgr.h"
 
 uint32 Increase_Level;
 uint32 MaxItemLevel = 80;
@@ -16,7 +16,9 @@ bool LevelItemAnnounce = true;
 class mod_levelitem_Conf : public WorldScript
 {
 public:
-    mod_levelitem_Conf() : WorldScript("mod_levelitem_Conf") { }
+    mod_levelitem_Conf() : WorldScript("mod_levelitem_Conf", {
+        WORLDHOOK_ON_BEFORE_CONFIG_LOAD
+    }) { }
 
     // Load Configuration Settings
     void OnBeforeConfigLoad(bool /*reload*/) override
@@ -32,14 +34,14 @@ class mod_levelitem_Announce : public PlayerScript
 {
 
 public:
-    mod_levelitem_Announce() : PlayerScript("mod_levelitem_Announce") {}
+    mod_levelitem_Announce() : PlayerScript("mod_levelitem_Announce", {
+        PLAYERHOOK_ON_LOGIN
+    }) {}
 
-    void OnLogin(Player* player) override
+    void OnPlayerLogin(Player* player) override
     {
         if (LevelItemAnnounce)
-        {
             ChatHandler(player->GetSession()).SendSysMessage("This server is running the |cff4CFF00Level Item |rmodule.");
-        }
     }
 };
 

